client/checks.go

```diff
@@ -100,3 +119,55 @@ func (client *Client) Checks(opts *ChecksOptions) ([]*CheckInfo, error) {
 	}
 	return checks, nil
 }
+
+// Start starts the checks named in opts.Names. We ignore ops.Level for this
+// action.
+func (client *Client) StartChecks(opts *ChecksOptions) (response string, err error) {
+	response, err = client.doMultiCheckAction("start", opts.Names)
+	return response, err
+}
+
+// Stop stops the checks named in opts.Names. We ignore ops.Level for this
+// action.
+func (client *Client) StopChecks(opts *ChecksOptions) (response string, err error) {
+	response, err = client.doMultiCheckAction("stop", opts.Names)
+	return response, err
+}
+
+type multiCheckActionData struct {
+	Action string   `json:"action"`
+	Checks []string `json:"checks"`
+}
+
+func (client *Client) doMultiCheckAction(actionName string, checks []string) (changeID string, err error) {
+	action := multiCheckActionData{
+		Action: actionName,
+		Checks: checks,
+	}
+	data, err := json.Marshal(&action)
+	if err != nil {
+		return "", fmt.Errorf("cannot marshal multi-check action: %w", err)
+	}
+	headers := map[string]string{
+		"Content-Type": "application/json",
```

Comment:
Oddly, we don't normally provide the content type for requests (even though it's almost always JSON, except for files, some of which are multipart). We do in `POST /v1/services` though, so I don't mind specifying it here for consistency with that.

I do wonder if we should get the client to add it by default (unless it's overridden in `Headers`). Would that cause any issues?

I realise this is not related to this PR.

Reply:
Open an issue for someone to make this change in the future. For now, since this isn't technically required and in the future it'll be the default, leave it out here.

Reply:
https://github.com/canonical/pebble/issues/561